packages/frontend/src/utils/bigNumberConfig.ts

```diff
@@ -1,12 +1,11 @@
 import { ApiQueryResults } from 'common';
 
 const bigNumberConfig = (data: ApiQueryResults | undefined) => {
-    if (data) {
-        const metric: string = data.metricQuery.metrics[0];
-        const bigNumber: number | string = data.rows[0][metric];
-        return bigNumber;
-    }
-    return '';
+    if (!data) return null;
```

Comment:
I think this will still fail if `data.rows = [[]]` because `data` is truthy and `data.rows` is truthy too.

Did this pass with empty data?